src/app/api/services/workflow/workflow.service.ts

```diff
@@ -19,7 +19,7 @@ export class WorkflowService {
      * @returns {Observable<SelectItem[]>}
      * @memberof WorkflowService
      */
-    get(): Observable<Workflow[]> {
+    get(): Observable<DotWorkflowActions[]> {
```

Comment:
This endpoint does return Workflows Schemes not actions... so we don't have to replace this.

The concept is (I think @jdotcms could explain this better)

Workflow Actions belongs Workflow Schemes

Reply:
Yes, the Workflow has WorkflowAction, this method return Workflow no WorkflowAction